src/main/java/com/linecorp/armeria/client/routing/ServerGroup.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.client.routing;
+
+import java.util.List;
+
+public interface ServerGroup<T extends Node> {
+    List<T> nodes();
+
+    default boolean isReplaceable() {
```

Comment:
We want user can control if the ServerGroup can replaceable,sometimes(like some nodes failed) we want re-register the same groupName with different ServerGroup.The default value is irreplaceable to avoid replace exist config accidently.